### PR TITLE
Put the navigation bar of the homepage at the bottom on mobile

### PR DESCRIPTION
**Pull request by AlexandreSi**
image

### CODE REVIEW

File: newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageMenu.js [hunk 1]
```diff
@@ -145,52 +127,13 @@ export const HomePageMenu = ({
...
-                getIcon={getIcon}
-                isActive={false}
-                hideLabel={isMobileOrSmallScreen}
-                id={id}
-              />
-            ))}
-          </Column>
-        </div>
-      </Paper>
+      <HomePageMenuBar
```
Comment (ClementPasteau) [inline newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageMenu.js:130]: great refactor 👍

File: newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageMenu.js [hunk 2]
```diff
@@ -2,37 +2,22 @@
...
 import WebIcon from '../../../UI/CustomSvgIcons/Web';
-import Sun from '../../../UI/CustomSvgIcons/Sun';
-import Store from '../../../UI/CustomSvgIcons/Store';
+import SunIcon from '../../../UI/CustomSvgIcons/Sun';
+import StoreIcon from '../../../UI/CustomSvgIcons/Store';
 import Preferences from '../../../UI/CustomSvgIcons/Preferences';
 import GDevelopGLogo from '../../../UI/CustomSvgIcons/GDevelopGLogo';
 import GDevelopThemeContext from '../../../UI/Theme/GDevelopThemeContext';
-import Paper from '../../../UI/Paper';
+import HomePageMenuBar from './HomePageMenuBar';
```
Comment (ClementPasteau) [inline newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageMenu.js:18]: should this be called a bar if the menu is on the left on desktop? (don't have a better name yet though...)

**Review by ClementPasteau — APPROVED** [review]
can't see much to say! nice!

**Comment by ClementPasteau** [comment]
Just to be sure, have you checked on different mobile sizes how it looks? (especially the smallest (320px) or tablet (768px)

File: newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageMenu.js (hunk 2 above)
Comment (4ian) [reply newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageMenu.js:18]: I think it's fine, the task *bar* on Windows used to be able to live on the left

**Comment by AlexandreSi** [comment]
> Just to be sure, have you checked on different mobile sizes how it looks? (especially the smallest (320px) or tablet (768px)

320px (horizontal scroll)
image

400px
image

450px
image

**Pull request merged**